app/models/case/base.rb

```diff
@@ -61,7 +61,7 @@ def self.searchable_document_tsvector
 
   attr_accessor :message_text
 
-  acts_as_gov_uk_date :received_date, :date_responded,
+  acts_as_gov_uk_date :received_date, :date_responded, :external_deadline,
```

Comment:
You can move this into ICO Overturned base class, you just need to include _all_ the dates that `acts_as_gov_uk_date`, until we fix that to be cumulative.

Reply:
As you saw, removing it from Case::Base also seems to remove it from everything that derives from Case::Base.  Probably dependent on load order.  Would be fixed by fixing gov_uk_date_fields gem to allow multiple definitions.